.github/workflows/helm-test.yml

```diff
@@ -121,6 +124,47 @@ jobs:
       - name: List Services
         run: |
           minikube kubectl -- get services
+      - name: Service healthcheck (:30888/restconf/operations)
+        run: |
+          minikube_ip=$(minikube ip)
+          curl --user admin:admin -H "Content-Type: application/json" --insecure http://$minikube_ip:30888/restconf/operations
+      - name: Uninstall deployment
+        run: |
+          helm del ${{ github.event.inputs.app-name }} --purge
+          sleep 25 # Wait for helm to del deployment
+      - name: Install app helm chart, CLUSTER
+        run: |
+          helm install ${{ env.app-aggregator-dir}}/${{ github.event.inputs.app-name }}-helm/helm/${{ github.event.inputs.app-name }}-helm --name ${{ github.event.inputs.app-name }} --set lighty.replicaCount=3 --set lighty.akka.isSingleNode=false
+          sleep 120 # Wait for app to start and form cluster
```

Comment:
Try to implement some healthcheck call, so we don't have to wait 2 minutes all the time (I guess installation is usually done much faster)

Reply:
Will be done in later changes